internal/datasource/config/config_writer_test.go

```diff
@@ -183,7 +195,7 @@ func TestDeleteFile(t *testing.T) {
 			err := fileCache.UpdateFileCache(ctx, test.fileCache)
 			require.NoError(t, err)
 			slog.Info("", "", &agentconfig)
```

Comment:
just noticed slog in tests here... no need to fix but should use https://pkg.go.dev/testing#T.Log instead. Another thing for coding standards